package.json

```diff
@@ -55,6 +56,7 @@
     "retry": "0.6.1",
     "strophe": "1.2.4",
     "strophejs-plugins": "0.0.7",
+    "styled-components": "1.3.0",
```

Comment:
This is a peer-dependency for @atlaskit/util-shared-styles, which is a dependency of @atlaskit/single-select. Without it explicitly added in package.json, the "make" command would fail to properly create the app bundle.